documentation/components/Carousel.mdx

```diff
@@ -410,3 +410,14 @@ hideControls prop removes the control buttons from view until hovered.
     <div style={{ backgroundColor: ' rgb(0,204,0)', width: '700px' }}>Slide 3</div>
   </Carousel>
 </Playground>
+
+## Show Half Next and Prev Slides with infinite using set width
```

Comment:
At a glance I'm noticing an issue with the drag functionality for this example, but tomorrow I'll take a closer look at the updates made and see what could be the cause